cmd/kueue/main.go

```diff
@@ -262,69 +253,38 @@ func setupControllers(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manag
 		}
 	}
 
-	manageJobsWithoutQueueName := cfg.ManageJobsWithoutQueueName
-
 	if failedWebhook, err := webhooks.Setup(mgr); err != nil {
 		setupLog.Error(err, "Unable to create webhook", "webhook", failedWebhook)
 		os.Exit(1)
 	}
 
 	opts := []jobframework.Option{
-		jobframework.WithManageJobsWithoutQueueName(manageJobsWithoutQueueName),
-		jobframework.WithWaitForPodsReady(waitForPodsReady(cfg)),
+		jobframework.WithManageJobsWithoutQueueName(cfg.ManageJobsWithoutQueueName),
+		jobframework.WithWaitForPodsReady(config.IsWaitForPodsReadyEnabled(cfg.WaitForPodsReady)),
```

Comment:
What about something like this?
I don't think we should need two separate functions to process the configuration.

```suggestion
		jobframework.WithWaitForPodsReady(cfg.WaitForPodsReady),
```

Reply:
It makes sense.

Reply:
Also I think we can apply the same way to `EnabledFrameworks`.

Reply:
Done.